### PR TITLE
Common logic for sidebar

### PR DESCRIPTION
**Pull request by hadleyshapland**
Created a commonlogic.html file to hold the sidebar and then used jquery to include it in all the pages. This will cut down code duplication and make maintenance easier.

### CODE REVIEW

File: portfolio/src/main/webapp/background.html [hunk 1]
```diff
@@ -5,26 +5,28 @@
     <title>Hadley Shapland</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+
+    <!-- include jquery -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+
+    <script>
+      $(function(){
```
Comment (jerrychangusa) [inline portfolio/src/main/webapp/background.html]: Could you move this logic to script.js to try to keep the Javascript and HTML separate? This should also help with reducing duplicated logic.

**Review by jerrychangusa — COMMENTED** [review]
Nice!

File: portfolio/src/main/webapp/background.html (hunk 1 above)
Comment (hadleyshapland) [reply portfolio/src/main/webapp/background.html]: Good point, just fixed it!

File: portfolio/src/main/webapp/background.html [hunk 2]
```diff
@@ -4,27 +4,22 @@
...
   <body>
-    <!-- SIDEBAR -->
-      <div class="sidenav">
-        <a href="index.html">Home</a>
-        <a href="background.html">Background</a>
-        <a href="experience.html">Experience</a>
-        <a href="funfacts.html">Fun Facts</a>
-    </div>
-    <!-- END SIDEBAR -->
+    <div id="commonlogic"></div>
```
Comment (jerrychangusa) [inline portfolio/src/main/webapp/background.html]: can we rename the id to something more descriptive (e.g. sidebar)?

File: portfolio/src/main/webapp/commonlogic.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```
Comment (jerrychangusa) [inline portfolio/src/main/webapp/commonlogic.html]: consider changing the file name to something more descriptive (e.g. sidebar, sidenav)

**Review by jerrychangusa — APPROVED** [review]
LGTM

File: portfolio/src/main/webapp/background.html (hunk 2 above)
Comment (hadleyshapland) [reply portfolio/src/main/webapp/background.html]: Done!

File: portfolio/src/main/webapp/commonlogic.html (hunk 3 above)
Comment (hadleyshapland) [reply portfolio/src/main/webapp/commonlogic.html]: Renamed to sidebar.html

**Review by ghandre — APPROVED** [review]
Nice refactor, looks good!

**Pull request merged**